pkg/spanconfig/spanconfigkvsubscriber/testdata/system_span_configs

```diff
@@ -0,0 +1,127 @@
+
+start
+----
+
+update
+upsert [a,c):A
+upsert [d,f):D
+----
+
+updates
+----
+/M{in-ax}
+[a,c)
+[d,f)
+
+store-reader key=a
+----
+conf=A
+
+store-reader key=e
+----
+conf=D
+
+store-reader key=g
+----
+conf=MISSING
+
+update
+upsert {entire-keyspace}:_CLUSTER
+----
+
+updates
+----
+/M{in-ax}
+
+store-reader key=a
+----
+conf=A_CLUSTER
+
+store-reader key=e
+----
+conf=D_CLUSTER
+
+# The system span config we set should still apply.
+store-reader key=g
+----
+conf=MISSING_CLUSTER
+
+# Update span configs that target tenant keyspaces and ensure handlers are
+# invoked correctly.
+update
+upsert {source=1, target=1}:_HOST
+upsert {source=1,target=20}:_SECONDARY
+upsert {source=10, target=10}:_TEN
+----
+
+# M{in-ax} corresponds to the system tenant targeting itself.
```

Comment:
Is this right? I thought the system tenant targeting itself would would only capture its own keyspace without going into the tenant's. Wasn't that the whole point of separating out the EntireKeyspaceTarget into its own thing?